packages/marqus-desktop/src/main/index.ts

```diff
@@ -23,7 +24,7 @@ let onDispose: () => Promise<void>;
 export async function main(): Promise<void> {
   try {
     const configFile = await getConfig();
-    const log = await getLogger(configFile, console);
+    logger.add(getFileTransport(configFile));
```

Comment:
Another benefit of this setup means we can immediately log to the console even if we haven't loaded the config file yet.